changelog.d/556.feature

```diff
@@ -0,0 +1 @@
+Display GitLab project paths with their true letter casing.
```

Comment:
Display where? And this might be a `misc` rather than a `feature`?

Reply:
In the widget's listing of connected projects & the bot's response to `gitlab project` (i.e. "Room configured to bridge $path").

Reply:
Also, I didn't mark this as `misc` because it's user-facing.  Is `misc` also for minor touchups, even if they are visible?

Reply:
Sticking with `feature` for now